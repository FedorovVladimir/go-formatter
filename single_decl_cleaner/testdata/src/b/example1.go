package main

const ( // want "incorrect single declaration style"
	c = "c"
)
