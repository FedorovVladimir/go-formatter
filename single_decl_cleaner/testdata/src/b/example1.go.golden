package main

const c = "c"
